src/langs/fr.json

```diff
@@ -32,6 +32,14 @@
     "There are no collections available.": "Aucune collection disponible.",
     "Category": "Categorie",
     "Tags": "Tags",
-    "Search collections using keywords and range options": "Rechercher des collections en utilisant des mots-clés et les champs possibles"
+    "Search collections using keywords and range options": "Rechercher des collections en utilisant des mots-clés et les champs possibles",
+    "someone's avatar": "Avatar de {{name}}",
```

Comment:
The fr and en files are not symmetric, I think you miss some translations in both files.